x-pack/test/functional_with_es_ssl/apps/cases/create_case_form.ts

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+import uuid from 'uuid';
+import { FtrProviderContext } from '../../ftr_provider_context';
+
+export default ({ getPageObject, getService }: FtrProviderContext) => {
+  describe('Create case', function () {
+    const common = getPageObject('common');
+    const find = getService('find');
+    const cases = getService('cases');
+
+    before(async () => {
+      await common.navigateToApp('cases');
+    });
+
+    after(async () => {
+      await cases.api.deleteAllCases();
+    });
+
+    it('creates a case from the stack managament page', async () => {
+      const caseTitle = 'test-' + uuid.v4();
+      await cases.common.createCaseFromCreateCasePage(caseTitle);
+      const title = await find.byCssSelector('[data-test-subj="header-page-title"]');
```

Comment:
Do you mind if we check the desc, the tags, the sync alert options, no connector, and the status that have the values that we expect after the creation of the case?

Reply:
There's no easy way to test "no connector"  selected. I added validations for description and tag.

Reply:
You check for the existence of
<img width="590" alt="Screenshot 2022-03-22 at 1 59 10 PM" src="https://user-images.githubusercontent.com/7871006/159477183-6fc14bcf-74e7-4e7e-9449-c46d0a502951.png">

Reply:
The data-test-subj of this is quite generic and not specific to "add connector".  I guess I can check if the button has the correct text.

Reply:
This will only appear if the none connector is selected so it is a good assumption (+ it checks the callout functionality) that the connector is indeed the "none". I like the idea with the button.